source/d_qgis_plugin.rst

```diff
@@ -81,15 +81,24 @@ Simulate
 
 The most used API options are included in the newest version of the plugin. Important consideration is a difference between API v1 and v3 how initial waterlevels, laterals and boundaries are handled. The current status is as follows:
 
-============================= ================= ================= ===============
-Forcings                        Spatialite          API             Live site
-============================= ================= ================= ===============
-Boundary conditions             v1, v3              v3              v1, v3
-Initial water level             v1, v3              v1, v3          v3
-Laterals  1D and 2D             v1                  v3              -
-DWF (inflow)                    v1                  v3              -
-============================= ================= ================= ===============
-
+============================= =========================== ===================================== ==================================
+Forcings                        Live site                  3Di API Client                       OpenAPI client
+============================= =========================== ===================================== ==================================
+Boundary conditions            SQLite                     SQLite                                SQLite, can be overwritten
```

Comment:
OpenAPI Client, SqLite can be overwritten, but in that case all boundary conditions (both 1D and 2D) need to be overwritten

Reply:
added under the table